### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hermancollin**
This PR aims to update the README according to the new template (https://github.com/neuropoly/axondeepseg/wiki/Template-for-Readme-of-new-models).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,55 @@
...
+git clone https://github.com/axondeepseg/default-TEM-model
+```
+
+### Get the data
+- git@data.neuro.polymtl.ca:datasets/data_axondeepseg_tem
+- commit c778a33323a6e6c9c5bf38bd1e8a7038686f3423
+
+```
+git clone git@data.neuro.polymtl.ca:datasets/data_axondeepseg_tem
+cd dataset-used-to-train-the-model
```
Comment (mariehbourget) [inline README.md]:
```suggestion
cd data_axondeepseg_tem
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,55 @@
...
 AxonDeepSeg default TEM model and testing image. This model is suited for a resolution of 0.01 micrometer per pixel and was trained on mouse brain data collected via a Transmission Electron Microscope (TEM).
 
+## Dependencies
+- [ivadomed](https://ivadomed.org/) commit: e6554281d07b5afef9e68ce8b02e86b02bd68363
+- [axondeepseg](https://axondeepseg.readthedocs.io/en/latest/) commit: 805868e39eddf620c9f3d60d313cadffb1121bfb
+
+## Segment (ADS)
+To segment an image using this model, use
+```
+axondeepseg -t TEM -i <IMG_PATH> -m <path_to_model_folder> -s <PIXEL_SIZE>
```
Comment (mariehbourget) [inline README.md]: In the SEM readme, we removed the `-m` option from the CLI line.
I'm not sure which one makes more sense but we should make the readme files consistent.

Comment (hermancollin) [reply README.md]: Ah yes forgot to remove it thanks for the catch

**Comment by mariehbourget** [comment]
Here is the version file for TEM
[version_info.zip](https://github.com/axondeepseg/default-TEM-model/files/8181539/version_info.zip)

**Comment by hermancollin** [comment]
> Here is the version file for TEM [version_info.zip](https://github.com/axondeepseg/default-TEM-model/files/8181539/version_info.zip)

Is it normal that this file doesn't have the training_sha256 field? The BF version file also doesn't have it.

**Comment by mariehbourget** [comment]
From what I understand, the CONFIG INPUT part of the version file is taken directly from the config file used when launching ivadomed [here](https://github.com/ivadomed/ivadomed/blob/d4181999c7dcda3d0ef22161eaf3e32bf6d92b9f/ivadomed/main.py#L351).

- If ivadomed is run with the original config file: there are no training sha in it.
- If ivadomed is run from the config file inside the model folder, the training sha are present and copied with all the other fields.

In the TEM and BF case, it seems I ran the tests from the original config files, so without training sha. As discussed in the last meeting, this is not ideal and would need some clean-up on the ivadomed side. However, for those particualr models, this is the best we have.

**Comment by hermancollin** [comment]
Version file added. Now this branch is consistent with the SEM model readme.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,47 @@
...
+
+### Get the data
+The TEM dataset used to train this model is hosted on git-annex at `data.neuro.polymtl.ca:datasets/data_axondeepseg_tem`. The specific dataset revision hash used for training is documented in the *version_info.log* file.
+
+### Train this model
+To train the model, please first update the following fields in the aforementioned JSON configuration file:
+- `gpu_ids`: specific to your hardware
+- `path_output`: where the model will be saved
+- `loader_parameters:path_data`: path to training data
+- `loader_parameters:bids_config`: path to the custom bids config located in `ivadomed/config/config_bids.json`
```
Comment (jcohenadad) [inline README.md:33]: why is this needed if a custom JSON is provided with the repository?

**Review by mariehbourget — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (hermancollin) [reply README.md:33]: Because the BIDS config is not provided with the models (different from the model JSON config)
